tests/RegisterAddres.test.js

```diff
@@ -0,0 +1,8 @@
+const {
+    ApiPromise,
+    WsProvider,
+    Keyring
+} = require("@polkadot/api");
```

Comment:
@nathanwhit this line makes Jest choke with 

```
Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.
```

Do you know which kind of syntax does Polkadot use (I think it's TypeScript) and where can I find the relevant configuration ?

Reply:
I haven't. I'll try copying all configuration from there.